package.json

```diff
@@ -41,7 +43,9 @@
     "xhr2": "0.1.3"
   },
   "devDependencies": {
+    "@peculiar/webcrypto": "^1.0.9",
     "eslint": "5.6.1",
+    "fast-text-encoding": "^1.0.0",
```

Comment:
Did this dependency get added by mistake?

Reply:
This was part of the shim for webcrypto in jest. I have removed it since webcrypto test is in karma now.